distribution/docs/src/main/resources/_contents/_catalog-frameworks/global-attribute-validators-definition-contents.adoc

```diff
@@ -55,7 +55,7 @@ The `validator` key must have a value of one of the following:
  - `futuredate`
  * `arguments`: (0) [NO ARGUMENTS]
  - `range`
- ** (2) [number (decimal or integer): inclusive lower bound, number (decimal or integer): inclusive upper bound]
+ ** (2) [number (decimal or integer): inclusive lower bound, number (decimal or integer): inclusive upper bound, lower bound must be greater than or equal to zero and the upper bound must be greater than or equal to the lower bound]
```

Comment:
This needs to go in `size` instead of `range`. Also this shouldn't go between the `[]` since this list is for the validator arguments (it could go after the list or in a nested bullet below it).